### ISSUE TITLE
Clicking on a user name leads you to an error page

### ISSUE THREAD

**Issue by gergely-ujvari** [hypothesis/h#2096]
Seen this on stage.hypothes.is and local env master branch

Clicking on the username on an annotation card is changed and sadly broken
Originally it lead you to the standalone page (if I remember of well) of that annotation. I'm remembering something wrong or has this been changed recently

Now it generates a link which would take you to the user stream: 
```
http://localhost:5000/u/acct:ujvari@localhost
https://stage.hypothes.is/u/acct:aron@stage.hypothes.is
``` 

but you'll get an error page
the cause is a server side exception (I think we have seen this on sentry):
```
2015-03-26 08:22:45,025 [17712] [h.views:ERROR] AttributeError: 'tuple' object has no attribute 'groups'
Traceback (most recent call last):
  File "/home/ujvari/Envs/h1/local/lib/python2.7/site-packages/pyramid/tweens.py", line 21, in excview_tween
    response = handler(request)
  File "/home/ujvari/Envs/h1/local/lib/python2.7/site-packages/pyramid_tm/__init__.py", line 94, in tm_tween
    reraise(*exc_info)
  File "/home/ujvari/Envs/h1/local/lib/python2.7/site-packages/pyramid_tm/__init__.py", line 75, in tm_tween
    response = handler(request)
  File "/home/ujvari/Envs/h1/local/lib/python2.7/site-packages/pyramid/router.py", line 163, in handle_request
    response = view_callable(context, request)
  File "/home/ujvari/Envs/h1/local/lib/python2.7/site-packages/pyramid/config/views.py", line 329, in attr_view
    return view(context, request)
  File "/home/ujvari/Envs/h1/local/lib/python2.7/site-packages/pyramid/config/views.py", line 305, in predicate_wrapper
    return view(context, request)
  File "/home/ujvari/Envs/h1/local/lib/python2.7/site-packages/pyramid/config/views.py", line 287, in _authdebug_view
    return view(context, request)
  File "/home/ujvari/Envs/h1/local/lib/python2.7/site-packages/pyramid/config/views.py", line 355, in rendered_view
    result = view(context, request)
  File "/home/ujvari/git/16/h/h/views.py", line 106, in stream
    if parts is not None and parts.groups()[1] == request.domain:
AttributeError: 'tuple' object has no attribute 'groups'
```

**Comment by gergely-ujvari** [comment]
I remember now. Redirecting you to the user stream is okay if you click on it on an annotation card. The error thrown sadly is still wrong.

**Comment by tilgovi** [comment]
Looks like @seanh fixed this in #2100
